telegram_dispatch.js

```diff
@@ -0,0 +1,73 @@
+const request = require('request');
+const express = require('express');
+const app = express();
+const TelegramBot = require('node-telegram-bot-api');
+const fs = require('fs');
+
+// const aws = require('aws-sdk');
+
+// let s3 = new aws.S3({
+//   accessKeyId: process.env.AWS_KEY,
+//   secretAccessKey: process.env.AWS_SECRET
+// });
+
+const sqs = require('sqs');
+
+const queue = sqs({
+   access: process.env.AWS_KEY,
+   secret: process.env.AWS_SECRET,
+   region: 'us-east-1' // defaults to us-east-1
+});
+const production_queue_name = "intelligenttrading-sqs-production"
+
+const token = process.env.TELEGRAM_BOT_TOKEN;
+const bot = new TelegramBot(token, { polling: false });
```

Comment:
Is this the only dispatcher for all the microservices?
Will we add a `onChartRequest` with a request to the image service?